tests/functional/spec/routing_and_skipping_section_dependencies.spec.js

```diff
@@ -0,0 +1,236 @@
+import AgePage from "../generated_pages/new_routing_and_skipping_section_dependencies/age.page";
+import HouseHoldPersonalDetailsSectionSummaryPage from "../generated_pages/new_routing_and_skipping_section_dependencies/household-personal-details-section-summary.page";
+import HouseholdSectionSummaryPage from "../generated_pages/new_routing_and_skipping_section_dependencies/household-section-summary.page";
+import ListCollectorAddPage from "../generated_pages/new_routing_and_skipping_section_dependencies/list-collector-add.page";
+import ListCollectorPage from "../generated_pages/new_routing_and_skipping_section_dependencies/list-collector.page";
+import NamePage from "../generated_pages/new_routing_and_skipping_section_dependencies/name-block.page";
+import PrimaryPersonSummaryPage from "../generated_pages/new_routing_and_skipping_section_dependencies/primary-person-summary.page";
+import ReasonNoConfirmationPage from "../generated_pages/new_routing_and_skipping_section_dependencies/reason-no-confirmation.page";
+import RepeatingAgePage from "../generated_pages/new_routing_and_skipping_section_dependencies/repeating-age.page";
+import RepeatingSexPage from "../generated_pages/new_routing_and_skipping_section_dependencies/repeating-sex.page";
+import SecurityPage from "../generated_pages/new_routing_and_skipping_section_dependencies/security.page";
+import SkipAgePage from "../generated_pages/new_routing_and_skipping_section_dependencies/skip-age.page";
+import SkipConfirmationPage from "../generated_pages/new_routing_and_skipping_section_dependencies/skip-confirmation.page";
+import SkipConfirmationSectionSummaryPage from "../generated_pages/new_routing_and_skipping_section_dependencies/skip-confirmation-section-summary.page";
+import SkipSectionSummaryPage from "../generated_pages/new_routing_and_skipping_section_dependencies/skip-section-summary.page";
+
+import HubPage from "../base_pages/hub.page";
+
+describe("Given the routing and skipping section dependencies questionnaire", () => {
+  beforeEach("Load the survey", () => {
+    browser.openQuestionnaire("test_new_routing_and_skipping_section_dependencies.json");
+  });
```

Comment:
This could be tested for both schemas without duplicating the test case.

Reply:
I think we need to be consistent as we haven't done this for any other new rules. We also are looking to deprecate the other rules not add more code. Because we have unit test coverage for both and are looking to be pragmatic with functional tests with the other points mentioned above I think we should leave as is